com.koushikdutta.inkwire/res/values-de/strings.xml

```diff
@@ -0,0 +1,45 @@
+<resources>
+    <string name="app_name">Inkwire</string>
+    <string name="welcome_to_inkwire">Willkommen bei Inkwire</string>
+    <string name="screenshare_in_progress">Dein Bildschirm wird übertragen.</string>
+    <string name="share_this_android">Übertrage deinen Bildschirm an einen anderen Nutzer</string>
+    <string name="share">Freigeben</string>
+    <string name="share_access_code">Zugangscode weitergeben</string>
+    <string name="see_shared_android">Betrachte den Bildschirm eines anderen Nutzers</string>
+    <string name="access">Verbinden</string>
+    <string name="share_access_code_info">Um die Übertragung zu beginnen, sende den folgenden Zugangscode an die Person, die Dir helfen soll.</string>
+    <string name="creating_access_code">Zugangscode erzeugen</string>
+    <string name="send_link">Link senden</string>
+    <string name="copy_link">Link kopieren</string>
+    <string name="or_send_link">Oder sende einen Zugangslink:</string>
+    <string name="or_invite_link">Oder sende einen Einladungslink, mit der Bitte den Bildschirm zu teilen:</string>
+    <string name="waiting_for_connection">Warte auf die Verbindung...</string>
+    <string name="link_copied">Link in die Zwischenablage kopiert</string>
+    <string name="allow_screen_capture">Screenshot erlauben</string>
+    <string name="remote_user_info">Der andere Nutzer ist in der Lage deinen Bilschirm zu sehen und darauf zu zeichnen. Du kannst die Verbindung jederzeit in der Benachrichtigungsleiste beenden.</string>
+    <string name="got_it">gefunden</string>
+    <string name="enable_the_microphone">Aktiviere das Mikrofon, um die Sprachverbindung zu starten.</string>
+    <string name="mute_chat">Stumm</string>
+    <string name="stop">Stop</string>
+    <string name="enter_access_code">Zugangscode eingeben</string>
+    <string name="enter_access_code_info">Bitte den Nutzer, auf dessen Android Du zugreifen möchtest, auf "Freigeben" zu klicken und dir den Zugangscode zu geben.</string>
+    <string name="access_code">Zugangscode</string>
+    <string name="creating_invite_link">Einladungslink erzeugen</string>
+    <string name="error_contacting_server">Es gab einen Fehler beim Verbinden zu den Inkwire Servern.</string>
+    <string name="connecting_dotdotdot">verbinde&#8230;</string>
+    <string name="you_were_invited">Ein anderer Inkwire Nutzer möchte seinen Bildschirm mit dir teilen.</string>
+    <string name="connect">Verbinden</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="session_not_found">Inkwire Sitzung nicht gefunden. Diese ist möglicherweise abgelaufen.</string>
+    <string name="connected_to_screenshare">Du siehst einen geteilten Bildschirm.</string>
```

Comment:
Sounds like: "You are viewing a split screen." 
Maybe something like "Du siehst einen mit Dir geteilten Bildschirm."? 
IDK, a matter of taste. 
And sorry for the commas. ;)